recipes/gw-sklearn-xarray/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "sklearn_xarray" %}
```

Comment:
Should this be "gw-sklearn-xarray"? Otherwise, this conflicts with the name on PyPI from the base of the fork.